src/LibChorus/LibChorusActivitySource.cs

```diff
@@ -0,0 +1,24 @@
+﻿// // Copyright (c) 2025-2025 SIL International
+// // This software is licensed under the MIT License (http://opensource.org/licenses/MIT)
```

Comment:
Is the double comment (`// //`) automatically generated by https://github.com/sillsdev/chorus/blob/master/Chorus.sln.DotSettings#L3-L5?

Reply:
not sure, it was added by my IDE, but it probably doesn't need the double comment